supervision/dataset/formats/pascal_voc.py

```diff
@@ -17,15 +17,17 @@ def object_to_pascal_voc(
     object_name = SubElement(root, "name")
     object_name.text = name
 
+    xyxy -= 1
```

Comment:
`-= 1` ?

Reply:
Just a minute, facing some IDE issues, some commits were not pushed

Reply:
Fixed